kodi_addon_checker/check_addon.py

```diff
@@ -155,16 +157,15 @@ def _check_for_invalid_json_files(report: Report, file_index):
                                   relative_path(path)))
 
 
-def _check_addon_xml(report: Report, addon_path):
+def _check_addon_xml(report: Report, addon_path, addon_xml):
```

Comment:
Let's keep the name consistent, either tree or addon_xml. Or go for something that better describes it, like xml_root?

Reply:
how about parsed_xml ?

Reply:
Fixed